package.json

```diff
@@ -7,7 +7,7 @@
     "build": "rm -rf dist && tsc",
     "codegen": "npm run codegen:graphql-types",
     "codegen:graphql-types": "graphql-codegen",
-    "codegen:openapi-types": "npm run lint-openapi && ts-node-esm -P scripts/tsconfig.openapi-typescript.json scripts/generateOpenAPITypes.mts",
+    "codegen:openapi-types": "openapi-typescript ./openapi.yml -o ./src/generated/openapi/types.ts",
```

Comment:
generateOpenAPITypes.mts only inserted some eslint ignore comments into the generated file. I chose to simply ignore generated code by eslint altogether.

I first tried to getting this script to work, by trying different extensions, using `ts-node`, porting the script to Javascript, and using different tsconfig. I'm sure it's possible, but removing it altogether seems a better solution anyway.